airflow/api_fastapi/core_api/routes/public/event_logs.py

```diff
@@ -126,21 +126,17 @@ def get_event_logs(
         base_select = base_select.where(Log.dttm > after)
     event_logs_select, total_entries = paginated_select(
         base_select,
-        [],
+        None,
```

Comment:
We probably could use keyword argment here like the change in `airflow/api_fastapi/core_api/routes/public/connections.py`?